service/context/standard_test.go

```diff
@@ -0,0 +1,337 @@
+package context_test
+
+import (
+	"errors"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"encoding/json"
+
+	"github.com/ant0ine/go-json-rest/rest"
+
+	"github.com/tidepool-org/platform/service"
+	"github.com/tidepool-org/platform/service/context"
+)
+
+type Stringer struct {
+	str string
+}
+
+func (s *Stringer) String() string { return s.str }
+
+var _ = Describe("Standard", func() {
+	Context("JSONResponse", func() {
+		Context("encoded as JSON", func() {
+			It("is an empty object if no fields are specified", func() {
+				jsonResponse := &context.JSONResponse{}
+				Expect(json.Marshal(jsonResponse)).To(MatchJSON("{}"))
+			})
+
+			It("is a populated object if fields are specified", func() {
+				jsonResponse := &context.JSONResponse{
+					Errors: []*service.Error{
+						{
+							Code:   "test-code",
+							Detail: "test-detail",
+							Status: 400,
+							Title:  "test-title",
+						},
+					},
+					Meta: &context.Meta{
+						Trace: &context.Trace{
+							Request: "test-request",
+							Session: "test-session",
+						},
+					},
+				}
+				Expect(json.Marshal(jsonResponse)).To(MatchJSON("{" +
```

Comment:
you should be able to do the JSON like so  `{"page": 1, "fruits": ["apple", "peach"]}`  - maybe this would clean it up a little?